src/ingest_validation_tools/schema_loader.py

```diff
@@ -46,9 +46,9 @@ class SchemaVersion:
     version: str = ""
     directory_path: Optional[Path] = None
     table_schema: Optional[str] = ""  # legacy only
-    path: Union[Path, str] = ""
-    contributors_paths: List[str] = field(default_factory=list)
-    antibodies_paths: List[str] = field(default_factory=list)
+    path: Path = Path()
+    contributors_schemas: List[SchemaVersion] = field(default_factory=list)
```

Comment:
Reformatted how antibodies/contributors references are tracked (list of paths -> list of schemas)